lib/tests/streamlit/web/server/component_request_handler_test.py

```diff
@@ -21,7 +21,7 @@
 from streamlit.web.server import ComponentRequestHandler
 
 URL = "http://not.a.real.url:3001"
-PATH = "not/a/real/path"
```

Comment:
commonpath will error if comparing absolute and relative paths. Both paths will always be relative, so I want the path in the test to be absolute.